searx/static/themes/simple/src/less/detail.less

```diff
@@ -67,7 +67,9 @@ article.result-images .detail {
     p {
       color: var(--color-result-detail-label-font);
       font-size: 0.9rem;
-      line-height: 0.9rem;
+      white-space: nowrap;
+      text-overflow: ellipsis;
+      overflow: hidden;
```

Comment:
While I was looking into this I fond an old issue .. here I removed the `line-height: 0.9rem;` since setting this value causes the lower part of letters such as g to be cut off:

Before ..

![image](https://github.com/searxng/searxng/assets/554536/00e56ccc-622f-476f-adde-97ebcf812312)


After removing line-height ..

![image](https://github.com/searxng/searxng/assets/554536/9e15b4ba-b47c-48e0-a517-1b257261d30c)

Reply:
Hmm, this change was meant to reduce the spacing between the individual entries/rows.
So now, the last entry is sometimes cut off a bit again.

I think we can achieve the same by using a negative margin, I'll have a look.

Reply:
Okay, `margin: 0.8rem 0;` fixes this (no cut off entries anymore and no text cut off).